styles/angular-patternfly.css

```diff
@@ -0,0 +1,29 @@
+.donut-chart-pf {
+  width: 100%;
+  float: left;
+  padding-top: 10px;
+}
+
+.donut-tooltip-pf {
+  background: #434343;
+  color: #fff;
+  opacity: 0.9;
+  filter: alpha(opacity=90);
+  padding: 2px 6px;
+}
+
+.donut-title-big-pf {
+  font-family: "Open Sans", Helvetica, Arial, sans-serif;
+  font-size: 22px;
+  font-weight: normal;
```

Comment:
We need to use font-weight: 400 instead of normal. The way the fonts are currently declared in the font file are a bit weird and normal isn't mapped to anything.

What is the reason for needing the specify the font-family? That should be on the body and picked up by default.